egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -245,6 +304,12 @@ def AddFinalLayer(config_lines, input, output_dim,
     prev_layer_output = AddAffineLayer(config_lines,
             final_node_prefix , input, output_dim,
             ng_affine_options)
+    if final_mixture_window_length is not None:
+        [prev_layer_output, cur_left_context, cur_right_context] = AddPerDimAffineLayer(config_lines,
```

Comment:
Also it would be convenient if you describe what the final_mixture_window is and why you are using it here. This will ensure that the reader will not have to rely on top level scripts to understand why you are doing this.